src/_P039_Thermosensors.ino

```diff
@@ -276,25 +293,23 @@ boolean Plugin_039(byte function, struct EventStruct *event, String& string)
 
       if (P039_MAX_TYPE == P039_MAX31856) {
 
+
+        // ensure MODE3 access to SPI device
+        SPI.setDataMode(SPI_MODE3);
+
         // init string - content accoring to inital implementation of P039 - MAX31856 read function
-        uint8_t sendBuffer[11] = {0x80, 0x01, (uint8_t) P039_TC_TYPE, 0xFF, 0x7F, 0xC0, 0x7F, 0xFF, 0x80, 0x00, 0x00 };
+        // write to Adress 0x80
+        // activate 50Hz filter in CR0, choose averaging and TC type from configuration in CR1, activate OV/UV/OC faults, write defaults to CJHF, CJLF, LTHFTH, LTHFTL, LTLFTH, LTLFTL, CJTO
```

Comment:
While we're at it... why not add a selector in the plugin for 50Hz/60Hz?

Reply:
Hm, my kind of thinking - already there. I was just keeping the init sequence "as is" for the moment...i hesitate to use the configuration values in the INIT section, as those could as well only be partly set. Is there a way to define the default value of the parameter box in the plugin configuration web page. Is it the first item of the list ?

Reply:
Solved in next push.

Reply:
The PLUGIN_INIT call is only done after loading the settings.
So the PLUGIN_INIT should only use the settings and not try to use some defaults and later correct it.

N.B. PLUGIN_INIT is NOT called if the task using that plugin is not enabled.